src/app/asign-family/asign-family.component.ts

```diff
@@ -124,6 +124,15 @@ export class AsignFamilyComponent implements OnInit, OnDestroy {
     if (this.identifier != this.familyLists.helper.phone.thePhone)
       this.identifier = this.familyLists.helper.phone.thePhone
   }
+
+  getKeyboardType() {
+    return sessionStorage.getItem('keyboardType') || 'text'
+  }
+
+  toggleKeyboardType() {
```

Comment:
It works for me without adding more logic for blur/focus of the keyboard, so lmk if it also works for you or should I add logic for it

Reply:
I tested it on my iphone - and the keyboard doesn't change (just the alignment changes) - can you try the blur / focus?